frontend/src/pages/HomePage.tsx

```diff
@@ -2,6 +2,7 @@ import {Workout} from "../types/Workout.ts";
 import {useEffect} from "react";
 import axios from "axios";
 import {Link} from "react-router-dom";
+import "/src/App.css"
```

Comment:
i think we should do the styling in a css-file for each component. in app.css there should only be stylings that adress everything - and i dont think, we need to import it then.

Reply:
You're right, that's not necessary.